cmd/lastZWriter/lastZWriter.go

```diff
@@ -87,6 +113,51 @@ func fastaFinder(lastZ string, pairwise string, reference string, species string
 	return theseLines
 }
 
+// fastaFinder_Simple has no matrix string
+// fastaFinder_Simple has a different output file name structure: ref.species/qName/tName.qName.axt
+func fastaFinder_Simple(lastZ string, pairwise string, reference string, species string, par string) (lines []string) {
+	var currLine string
+	var theseLines []string
+	var tMatches, qMatches, tFiles, qFiles []string
+	tPath := pairwise + "/" + reference + ".byChrom"
+	qPath := pairwise + "/" + species + ".byChrom"
+
+	if _, e := os.Stat(tPath); os.IsNotExist(e) {
+		log.Fatalf("There is no .byChrom directory for the target (reference) species.")
+	}
+	if _, e := os.Stat(qPath); os.IsNotExist(e) {
+		log.Fatalf("There is no .byChrom directory for the query species.")
+	}
+
+	tMatches, _ = filepath.Glob(tPath + "/*.fa")
+	qMatches, _ = filepath.Glob(qPath + "/*.fa")
+
+	for tF := range tMatches {
+		_, tName := filepath.Split(tMatches[tF])
+		tFiles = append(tFiles, tName)
+	}
+	for qF := range qMatches {
+		_, qName := filepath.Split(qMatches[qF])
+		qFiles = append(qFiles, qName)
+	}
+	for t := range tFiles {
+		tName := strings.TrimSuffix(tFiles[t], ".fa")
+		for q := range qFiles {
+			qName := strings.TrimSuffix(qFiles[q], ".fa")
+			// currLine reflects that fastaFinder_Simple has no matrix string
+			// currLine also reflects that fastaFinder_Simple has a different output file name structure: ref.species/qName/tName.qName.axt
+			currLine = lastZ + " " + pairwise + "/" + reference + ".byChrom" + "/" + tFiles[t] + " " + pairwise + "/" + species + ".byChrom" + "/" + qFiles[q] + " --output=" + pairwise + "/" + reference + "." + species + "/" + qName + "/" + tName + "." + qName + ".axt" + " --action:target=multiple" + " --format=axt " + par
```

Comment:
> I'd suggest using a more programmatic way to build this string. The `filepath` package will join the strings like you've done manually, however the advantage here is the package will build filename paths in a way compatible with the target operating system-defined file paths (ie linux, windows, mac, etc.)
> 
> * forward and backward slashes to the target operating system. Windows paths use `\` while most others use `/`
>   You would enter all your separate strings like this:
> 
> If you go this route, don't forget to import the package at the top:
> 
> ```go
> import (
>     "path/filepath"
> )
> ```

I am making a note to edit this a little bit because currLine is not 1 path but a string containing many paths